docker-compose.link.yaml

```diff
@@ -13,7 +13,17 @@ services:
         npm run start:watch"
     networks:
       - ffc-demo
+      - default
+    ports:
+      - "3003:3003"
+
+  ffc-demo-claim-postgres:
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
```

Comment:
The `postgres_data` volume can be removed from `docker-compose.link.yaml` as well since it's now back in the base `docker-compose.yaml`.

Reply:
Nice catch, thank you.